app/models/core_data_connector/item.rb

```diff
@@ -9,7 +11,65 @@ class Item < ApplicationRecord
     include UserDefinedFields::Fieldable
     include Search::Item
 
+    after_create :fcc_import
```

Comment:
This looks good, I like the simplicity of the approach. Could we extract all of the FairCopy.cloud specific logic to its own module?